src/mono/wasm/debugger/DebuggerTestSuite/Inspector.cs

```diff
@@ -388,11 +390,15 @@ public async Task OpenSessionAsync(Func<InspectorClient, CancellationToken, List
                         _logger.LogError($"Command {cmd_name} failed with {completedTask.Exception}. Remaining commands: {RemainingCommandsToString(cmd_name, init_cmds)}.");
                         throw completedTask.Exception!;
                     }
+
                     await Client.ProcessCommand(completedTask.Result, _cancellationTokenSource.Token);
                     Result res = completedTask.Result;
+
                     if (!res.IsOk)
                         throw new ArgumentException($"Command {cmd_name} failed with: {res.Error}. Remaining commands: {RemainingCommandsToString(cmd_name, init_cmds)}");
 
+                    if (DebuggerTestBase.RunningOnChrome && cmd_name == "Debugger.enable")
```

Comment:
It feels a little odd to pass an empty string just to ignore it unless it is chrome and is there ever a case where we'd try to navigate to "" on chrome?

Reply:
I'm not sure if I understood the question.
Before this change we startup the chrome passing the url that it should navigate. And on my tests I saw that sometimes it was timing out the debugger.enable command. So I decided to first startup the chrome navigating to about:blank then send the Debugger.enabled and then when we receive the answer we send the Page.navigate to the wasm page.
On firefox we don't need to navigate to the page because we are still starting the browser passing the wasm page as parameter.